tests/EppoClientTest.php

```diff
@@ -54,7 +54,7 @@ public function testGracefulModeDoesNotThrow()
         $mockLogger = $this->getMockBuilder(LoggerInterface::class)->getMock();
 
         $subjectAttributes = [['foo' => 3]];
-        $client = EppoClient::createTestClient($mockConfigRequester, $pollerMock, $mockLogger);
+        $client = EppoClient::createTestClient($mockConfigRequester, $pollerMock, $mockLogger, true);
```

Comment:
changed the default for `isGracefulMode` to `false`

Reply:
Good catch!